.github/workflows/update-gpuci.yml

```diff
@@ -13,19 +13,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Parse current axis YAML
+      - name: Fetch current RAPIDS versions from build matrix
+        id: parse_yaml
         uses: the-coding-turtle/ga-yaml-parser@v0.1.2
         with:
-          file: ci/axis/dask.yaml
-          export_to_envs: true
-          return_to_outputs: false
+          file: .github/workflows/build.yml
+          export_to_envs: false
+          return_to_outputs: true
```

Comment:
This was the only use of `ci/axis.dask.yaml`.

```shell
git grep axis
```

That file should be deleted as part of this PR.

Reply:
Probably should've provided more clarity around this in the PR description, but this file is actually getting pointed to by some of our Jenkins infra, which I'm assuming would start failing if we deleted this file:

https://github.com/rapidsai/gpuci-scripts/blob/aef3a3321f0393b2473214235b0c7b6e3950b1db/seed-job-dsl/jobs-other/dask/dask-build-environment/branch/dask_build_env_main.groovy#L1

My thinking was that we would:

- merge this in, ensure that the GHA image publishing is working as expected alongside the Jenkins job
- make the necessary internal changes to drop the scheduled Jenkins runs, which should allow us to delete the above linked file
- drop the contents of `ci/**` in this repo as a follow up once the Jenkins runs have been turned off

Does that seem like a reasonable approach? Happy to do things a different way, and/or add more writing here to clarify the other ops work that should be considered a follow up to this PR.

Reply:
ooooo thank you for that explanation! Yes I totally support that plan, I'd forgotten about the Jenkins half of this.